db/seeds.rb

```diff
@@ -109,9 +67,9 @@
                           shared_bathroom: true,
                           user_id: 1
                           )
-
-# listing1.categories << categories[4]
-# listing1.save
+li = ListingImage.create(image: File.open(File.join(Rails.root, 'app/assets/images/denver.jpg')))
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.